### PR TITLE
add experts index entries for deepfreeze, frozen modules and subinterpreters  

### PR DESCRIPTION
**Pull request by kumaraditya303**

### CODE REVIEW

File: core-developers/experts.rst [hunk 1]
```diff
@@ -286,6 +286,7 @@ Tool                Maintainers
 ==================  ===========
 Argument Clinic     larryhastings
 PEG Generator       gvanrossum, pablogsal, lysnikolaou
+Deepfreeze          kumaraditya303
```
Comment (hugovk) [inline core-developers/experts.rst:289]: Let's keep alphabetical order:
```suggestion
Deepfreeze          kumaraditya303
PEG Generator       gvanrossum, pablogsal, lysnikolaou
```

**Review by hugovk — COMMENTED** [review]
Thank you!

If you like, you can add an asterisk after your name, but it's completely optional:

> Unless a name is followed by a ‘\*’, you should never assign an issue to that person. Names followed by a ‘\*’ may be assigned issues involving the module or topic.

File: core-developers/experts.rst (hunk 1 above)
Comment (kumaraditya303) [reply core-developers/experts.rst]: Good idea done!

**Comment by kumaraditya303** [comment]
> If you like, you can add an asterisk after your name, but it's completely optional

Yeah I read that but I usually assign issues to myself after I read through the bug report so I'll not add asterisk.

**Review by ezio-melotti — APPROVED** [review]

**Review by hugovk — APPROVED** [review]

File: core-developers/experts.rst [hunk 2]
```diff
@@ -334,6 +335,7 @@ documentation       ezio-melotti, merwok, JulienPalard, willingc, hugovk
 emoji               Mariatta
 extension modules   encukou, ncoghlan
 filesystem          giampaolo
+frozen modules      kumaraditya303
```
[the remark this answers is not in the document]
Comment (gvanrossum) [reply core-developers/experts.rst:338]: You can add me too, and yes, Eric Snow too.

File: core-developers/experts.rst [hunk 3]
```diff
@@ -355,6 +357,7 @@ release management  tarekziade, malemburg, benjaminp, warsaw,
                     gvanrossum, anthonybaxter^, merwok, ned-deily,
                     birkenfeld, JulienPalard
 str.format          ericvsmith*
+subinterpreters     kumaraditya303
```
Comment (gvanrossum) [inline core-developers/experts.rst:360]: Here you should definitely add Eric Snow

File: core-developers/experts.rst [hunk 4]
```diff
@@ -355,6 +357,7 @@ release management  tarekziade, malemburg, benjaminp, warsaw,
...
 testing             voidspace, ezio-melotti
```
Comment (gvanrossum) [inline core-developers/experts.rst:361]: I don't think that voidspace (Michael Foord) is active any more. I'll ask him.

File: core-developers/experts.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (kumaraditya303) [reply core-developers/experts.rst]: Added.

File: core-developers/experts.rst (hunk 3 above)
Comment (kumaraditya303) [reply core-developers/experts.rst]: Added

File: core-developers/experts.rst [hunk 5]
```diff
@@ -285,6 +285,7 @@ Tools
 Tool                Maintainers
 ==================  ===========
 Argument Clinic     larryhastings
+Deepfreeze          ericsnowcurrently, gvanrossum, kumaraditya303
```
Comment (gvanrossum) [inline core-developers/experts.rst:288]: Not Eric here

Comment (CAM-Gerlach) [reply core-developers/experts.rst:288]: So then

```suggestion
Deepfreeze          ericsnowpreviously, gvanrossum, kumaraditya303
```

Comment (kumaraditya303) [inline core-developers/experts.rst:288]:
```suggestion
Deepfreeze          gvanrossum, kumaraditya303
```

**Review by gvanrossum — APPROVED** [review]

**Pull request merged**